package.json

```diff
@@ -38,14 +40,19 @@
     "react": "18.3.1",
     "react-dom": "18.3.1",
```

Comment:
One observation here is this package is using react 18.3.1 version and I would except CCM to upgrade to same version? currently CCM is using react 18.2.0